Sources/TVCommanderKit/TVCommanderKit+Models.swift

```diff
@@ -217,10 +217,13 @@ public struct TVRemoteCommand: Codable {
         /// Enum representing the keys on a TV's remote control
         public enum ControlKey: String, Codable {
             case powerOff = "KEY_POWEROFF"
+            case powerToggle = "KEY_POWER"
             case up = "KEY_UP"
             case down = "KEY_DOWN"
             case left = "KEY_LEFT"
             case right = "KEY_RIGHT"
+            case channelUp = "KEY_CHUP"
+            case channelDown = "KEY_CHDOWN"
```

Comment:
please move `channelUp` and `channelDown` to be listed right before `channelList`